pkg/services/rpcsrv/params/param.go

```diff
@@ -34,6 +34,13 @@ type (
 		Type  smartcontract.ParamType `json:"type"`
 		Value Param                   `json:"value"`
 	}
+
+	// FuncParamPair represents a pair of function argument parameters
+	// a slice of which is stored in FuncParam of smartcontract.MapType type.
+	FuncParamPair struct {
```

Comment:
I see you're roughly following `smartcontract` naming, but `Pair` is somewhat misleading here to me. It's key-value, KV or MapItem.

Reply:
KV, fixed.